src/main/scala/no/ndla/imageapi/controller/ImageController.scala

```diff
@@ -55,6 +55,25 @@ trait ImageController {
         pathParam[String]("image_id").description("Image_id of the image that needs to be fetched."))
         )
 
+    val getImageFile =
+      (apiOperation[ImageMetaInformation]("getImageFile")
+        summary "Fetches a raw image"
+        notes "Fetches an image with options to resize and crop"
+        parameters(
+        headerParam[Option[String]]("X-Correlation-ID").description("User supplied correlation-id. May be omitted."),
+        headerParam[Option[String]]("app-key").description("Your app-key. May be omitted to access api anonymously, but rate limiting may apply on anonymous access."),
+        pathParam[String]("filename").description("The image filename"),
```

Comment:
Hmm, tror ikke `url` er veldig presist; parameteret er jo en del av hele urlen (navnet på bildefilen (for eksempel `image.jpg` eller `norge.jpg`)). Hva med `urn`?